facets.toml

```diff
@@ -0,0 +1,3 @@
+[[facets]]
```

Comment:
@sarahemlin I think we should remove this file as it doesn't correspond to any of the entries in the csv

Reply:
Removed!  Ready to merge.